ahrefs/monorobot
# Adjusted label-matching to be case insensitive

## Description of the task

Adjusted label-matching to be case insensitive.

## How to test

In mock_payloads, one can find a newly added .json files called ```pull_request.closed_capitalized.json```.

To test that label-matching is now case-insensitive, run the following commands:
```
./notabot check --json path/to/file
```
The expected output should be:

```
Using push routing:  
  backend/api* -> #aa
  backend/megaindex* or backend/ahrefskit* -> #backend
  any -> #all-push-events
Using pull request routing:
  backend* -> #backend
  aa* -> #aa-git
  siren* -> #siren
  not backend* or aa* or siren* -> #frontend-bot
```
As the label matching is now case-insensitive and the PR in this test has label "AA", one should expect the notabot to send notifications to #aa-git. If label matching is still case-sensitive, notabot will send notifications to #frontend-bot.
